### PR TITLE
QAE-296 - E2E test case for related post structure

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for related post structure viz title and meta

### Screenshots

https://share.getcloudapp.com/Bludy8l0

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js:1]: need to update util for "publish-post"

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Single post font option under the customizer', () => {
+	it( 'related post structure option should apply correctly', async () => {
+		const relatedpostStructure = {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js:5]: constant name should be as "relatedPostStructure"

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		} );
+		await publishPost();
+
+		await createNewPost( {
+			postType: 'post',
+			title: 'test-post',
+
+		} );
+		await publishPost();
+		await page.goto( createURL( 'sample-post' ), {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js:25]: @SimranChougule1  getting error while redirecting to created post. Can you please check?

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js]: Done

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js (hunk 3 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js]: Resolved

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { createURL, createNewPost } from '@wordpress/e2e-test-utils';
+import { publishPost } from '../../../../utils/publish-post';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Single post option under the customizer', () => {
+	it( 'related post structure option should apply correctly', async () => {
+		const relatedPostStructure = {
+			'enable-related-posts': 1,
+			'related-posts-structure': {
+				'title-meta': 1,
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js:9]: the Title is not shown in the related posts in the chrome preview, once verify

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/blog/single-post/related-post-structure.test.js:9]: @SimranChougule1 can you please take a look into this

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**